docs/cinder-csi-plugin/using-cinder-csi-plugin.md

```diff
@@ -41,114 +44,119 @@
 
 # CSI Cinder driver
 
-## Kubernetes
+The Cinder CSI Driver is a CSI Specification compliant driver used by Container Orchestrators to manage the lifecycle of Openstack Cinder Volumes.
 
-### Compatibility
+## CSI Compatibility
 
-CSI version | Cinder CSI Plugin Version | Kubernetes Version
-:------     | :------------             | :-----------
-v1.1.0 |  v1.0.0, v1.1.0  docker image: k8scloudprovider/cinder-csi-plugin:latest | v1.15+
-v1.0.0 |  v1.0.0  docker image: k8scloudprovider/cinder-csi-plugin:v1.14 | v1.13, v1.14
-v0.3.0 |  v0.3.0 docker image: k8scloudprovider/cinder-csi-plugin:1.13.x| v1.11, v1.12, v1.13
-v0.2.0 |  v0.2.0 docker image: k8scloudprovider/cinder-csi-plugin:v0.2.0 | v1.10, v1.9
-v0.1.0 |  v0.1.0 docker image: k8scloudprovider/cinder-csi-plugin:v0.1.0| v1.9
+This plugin is compatible with CSI versions v1.2.0 , v1.1.0, and v1.0.0
 
-For sidecar version compatibility , please refer compatibility matrix for each sidecar here -
-https://kubernetes-csi.github.io/docs/sidecar-containers.html
+## Downloads
 
-### Requirements
+Stable released version images of the plugin can be found at [DockerHub](https://hub.docker.com/r/k8scloudprovider/cinder-csi-plugin)
 
-```
-RUNTIME_CONFIG="storage.k8s.io/v1=true"
-```
-MountPropagation requires support for privileged containers. So, make sure privileged containers are enabled in the cluster.
+## Kubernetes Compatibility
 
-Check [kubernetes CSI Docs](https://kubernetes-csi.github.io/docs/) for flag details and latest update.
+For each kubernetes official release, there is a corresponding release of Cinder CSI driver which is compatible with k8s release. It is recommended to use the corresponding version w.r.t kubernetes.
 
-> NOTE: All following examples need to be used inside instance(s) provisoned by openstack, otherwise the attach action will fail due to fail to find instance ID from given openstack cloud.
+For sidecar version compatibility with kubernetes, please refer [CompatibilityMatrix](https://kubernetes-csi.github.io/docs/sidecar-containers.html) of each sidecar.
 
-### Example local-up-cluster.sh
+## Driver Config
 
-```ALLOW_PRIVILEGED=true RUNTIME_CONFIG="storage.k8s.io/v1=true" LOG_LEVEL=5 hack/local-up-cluster.sh```
+Implementation of `cinder-csi-plugin` relies on following OpenStack services.
 
-### Deploy
+| Service                        | API Version(s) | Deprecated | Required |
+|--------------------------------|----------------|------------|----------|
+| Identity (Keystone)            | v2             | Yes        | No       |
```

Comment:
we actually have nova v2 (current version is v2.1 microversion)
do we want to list deprecated one here or just go with v3 keystone?

Reply:
@jichenjc Its just supported major versions I have added. Authenticating through keystone v2 API is still supported through our code , not removed completely, even though we don't recommend. So added here as deprecated but still can use it.

Reply:
ok, works for me ,thanks~